examples/plugin/com_osvr_example_Tracker.cpp

```diff
@@ -0,0 +1,122 @@
+/** @date 2015
```

Comment:
I think someone copied and pasted... unless you started this three months ago...